packages/service-library/src/servicelib/rabbitmq/_rpc_router.py

```diff
@@ -64,15 +64,17 @@ async def _wrapper(*args, **kwargs):
                         ):
                             raise
 
-                        _logger.exception("Unhandled exception:")
+                        _logger.exception(
+                            "Unhandled exception: %s", exc  # noqa: TRY401
```

Comment:
`"Unhandled exception"` is not saying anything useful ... and `"Unhandled exception: %s"` is still not useful and in addition redundant ;-)

There the idea is to have some message with some information. Please do read the warnings https://docs.astral.sh/ruff/rules/verbose-log-message/ before marking them as ignored ;-)

Reply:
I will revert this one. I got confused by the one below. It is not necessary to add any changes here.
As we discussed `Unhandled expiation` is enough since you can track back this comes form the RPC layer. So will not be adding anything else.
I have double checked in the logs